### PR TITLE
upd: tr.m3u live broadcasts have been updated. 

### PR DESCRIPTION
**Pull request by ramazansancar**
upd: corrected missing and corrupt publications or added alternative publications.
fix: Youtube hls links have been updated.
upd: Incorrect live stream sizes have been updated.

Missing broadcast links on https://www.artidijitalmedya.com/ have been added.

### CODE REVIEW

**Comment by ramazansancar** [comment]
I need help with something.

Some of the "tvg-id" values ​​have not been added to iptv-org before. Is it possible to add them collectively to https://github.com/iptv-org/database by opening a PR or does each one have to be opened as a separate issue?

Thanks :)

File: streams/tr.m3u [hunk 1]
```diff
@@ -264,8 +304,12 @@ https://edge1.socialsmart.tv/muglaturk/bant1/playlist.m3u8
 http://broadcasttr.com:1935/naturaltv/bant1/playlist.m3u8
 #EXTINF:-1 tvg-id="NaturalTV.tr",Natural TV (720p) [Not 24/7]
 https://edge1.socialsmart.tv/naturaltv/bant1/playlist.m3u8
+#EXTINF:-1 tvg-id="NoraTV.tr",Nora TV (1080p)
+https://live.artidijitalmedya.com/artidijital_noratv/noratv/playlist.m3u8
 #EXTINF:-1 tvg-id="NTV.tr",NTV (720p) [Not 24/7]
 https://dogus-live.daioncdn.net/ntv/ntv.m3u8
+#EXTINF:-1 tvg-id="NTV.tr",NTV (720p) [Not 24/7]
+https://dogus.daioncdn.net/ntv/ntv.m3u8?ce=3
```
Comment (Alstruit) [inline streams/tr.m3u:312]:
```suggestion
```
Duplicate

File: streams/tr.m3u [hunk 2]
```diff
@@ -246,15 +282,19 @@ http://lalegultv.netmedya.net/hls/lalegultv.m3u8
...
 https://med-musik.icdndhcp.com/live3/live3.m3u8
 #EXTINF:-1 tvg-id="MeltemTV.tr",Meltem TV (1080p)
 https://vhxyrsly.rocketcdn.com/meltemtv/playlist.m3u8
-#EXTINF:-1 tvg-id="MercanTV.tr",Mercan TV (720p) [Not 24/7]
+#EXTINF:-1 tvg-id="MercanTV.tr",Mercan TV (576p) [Not 24/7]
+https://live.artidijitalmedya.com/artidijital_mercantv/mercantv/playlist.m3u8
+#EXTINF:-1 tvg-id="MercanTV.tr",Mercan TV (576p) [Not 24/7]
+https://waw1.artiyerelmedya.net/mercantv/bant1/playlist.m3u8
+#EXTINF:-1 tvg-id="MercanTV.tr",Mercan TV (576p) [Not 24/7]
 https://waw2.artiyerelmedya.net/mercantv/bant1/playlist.m3u8
```
Comment (Alstruit) [inline streams/tr.m3u:298]:
```suggestion
```
Duplicate.

File: streams/tr.m3u [hunk 3]
```diff
@@ -74,6 +82,10 @@ https://cdn-cekmekoybeltv.yayin.com.tr/cekmekoybeltv/cekmekoybeltv_1080p/playlis
 http://stream.taksimbilisim.com:1935/ciftcitv/smil:ciftcitv.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="CiftciTV.tr",Çiftçi TV (720p) [Not 24/7]
 https://live.artidijitalmedya.com/artidijital_ciftcitv/ciftcitv/playlist.m3u8
+#EXTINF:-1 tvg-id="CiftciTV.tr",Çiftçi TV (1080p) [Not 24/7]
+https://ythls.armelin.one/channel/UCtzcebnk0sLhCBtJy7GJaCg.m3u8
```
Comment (Alstruit) [inline streams/tr.m3u:86]: Should be in streams/us_ythls.m3u.

File: streams/tr.m3u [hunk 4]
```diff
@@ -88,7 +100,11 @@ https://603c568fccdf5.streamlock.net/live/dhaweb1_C5efC/playlist.m3u8
 https://live.artidijitalmedya.com/artidijital_dimtv/dimtv/playlist.m3u8
 #EXTINF:-1 tvg-id="DiyanetTV.tr",Diyanet TV (1080p)
 https://eustr73.mediatriple.net/videoonlylive/mtikoimxnztxlive/broadcast_5e3bf95a47e07.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="DiyarTV.tr",Diyar TV (1080p)
+#EXTINF:-1 tvg-id="DiyanetTV.tr",Diyanet TV (1080p)
+https://ythls.armelin.one/channel/UC3d1AdmvAP6Jq16-WpD086g.m3u8
```
Comment (Alstruit) [inline streams/tr.m3u:104]: Should be in streams/us_ythls.m3u.

File: streams/tr.m3u [hunk 5]
```diff
@@ -146,17 +164,21 @@ https://win8.yayin.com.tr/haber61tv/smil:haber61tv.smil/playlist.m3u8
 https://haberglobaldvr.blutv.com/blutv_haberglobal_dvr/live.m3u8
 #EXTINF:-1 tvg-id="HaberGlobal.tr",Haber Global (720p) [Not 24/7]
 https://tv.ensonhaber.com/haberglobal/haberglobal.m3u8
+#EXTINF:-1 tvg-id="HaberGlobal.tr",Haber Global (1080p)
+https://ythls.armelin.one/channel/UCtc-a9ZUIg0_5HpsPxEO7Qg.m3u8
```
Comment (Alstruit) [inline streams/tr.m3u:168]: Should be in streams/us_ythls.m3u.

File: streams/tr.m3u [hunk 6]
```diff
@@ -313,10 +357,16 @@ https://livetv.powerapp.com.tr/pturkslow/slow.smil/playlist.m3u8
...
 #EXTINF:-1 tvg-id="PowerTV.tr",Power TV (1080p)
 https://livetv.powerapp.com.tr/powerTV/powerhd.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="PowerTV.tr",Power TV (720p)
+https://live.artidijitalmedya.com/artidijital_powertv/powertv/playlist.m3u8
 #EXTINF:-1 tvg-id="PowerTurkTV.tr",PowerTurk TV
 https://powerturk.blutv.com/blutv_powerturk/powerturk_sd.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="PowerTurkTV.tr",PowerTürk TV (720p) [Geo-blocked]
+https://waw1.artiyerelmedya.net/powerturktv/bant1/playlist.m3u8
+#EXTINF:-1 tvg-id="PowerTurkTV.tr",PowerTürk TV (720p) [Geo-blocked]
 https://waw2.artiyerelmedya.net/powerturktv/bant1/playlist.m3u8
```
Comment (Alstruit) [inline streams/tr.m3u:367]:
```suggestion
```
Duplicate.

File: streams/tr.m3u (hunk 3 above)
Comment (ramazansancar) [reply streams/tr.m3u:86]: Why should an institution broadcasting in Turkish in Turkey be in us.m3u?

**Comment by ramazansancar** [comment]
Disagreements were resolved. Can I request a PR review again? @Alstruit 

Thanks

**Review by Alstruit — APPROVED** [review]

**Comment by Dum4G** [comment]
> Contributor
> Author
> ###
> ** [ramazansancar](/ramazansancar) ** c

needs separate issue to be created for each missing channel which is currently listed below
image

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**